src/ui/simulator/cmake/components-htmllistbox.cmake

```diff
@@ -28,8 +28,6 @@ add_library(libantares-ui-component-htmllistbox
 		toolbox/components/htmllistbox/item/pathlistitem.cpp
 		toolbox/components/htmllistbox/datasource/datasource.h
 		toolbox/components/htmllistbox/datasource/datasource.hxx
-		toolbox/components/htmllistbox/datasource/areas.h

```

Comment:
Nice to have spotted that these 2 files are useless !

Reply:
Too bad I only noticed after re-factoring them...